Makefile

```
@@ -83,22 +82,23 @@ $(WWW_DIR):
 # Tests
 #
 
-test: unittest testmanual
+test: test-style unittest testmanual
 
-
-ut-%: build
-	@PYTHONPATH=.:$(PYTHONPATH) $(NOSETESTS) nibabel/tests/test_$*.py
-
-
-unittest: build
-	@PYTHONPATH=.:$(PYTHONPATH) $(NOSETESTS) nibabel --with-doctest
+unittest: build test-clean
+	export CHECK_TYPE=test; ./tools/ci/check.sh
 
 testmanual: build
-	@cd doc/source && PYTHONPATH=../..:$(PYTHONPATH) $(NOSETESTS) --with-doctest --doctest-extension=.rst . dicom
+	export CHECK_TYPE=doc; ./tools/ci/check.sh
 
+coverage: unittest
 
-coverage: build
-	@PYTHONPATH=.:$(PYTHONPATH) $(NOSETESTS) --with-coverage --cover-package=nibabel
+.PHONY: test-style
+test-style:
+	export CHECK_TYPE=style; ./tools/ci/check.sh
```

I would rather put `make style` in `check.sh` than call `check.sh` from here. And likewise with other tests.

```suggestion
.PHONY: style
style:
	flake8
```